tests/manual-test-cases/Group18-VIC-UI/setup-testbed.robot

```diff
@@ -236,7 +236,7 @@ Deploy Nimbus ESXi Server For NGC Testing
     [Return]  ${user}-${name}  ${ip}
 
 Deploy Nimbus vCenter Server For NGC Testing
```

Comment:
If the name field can be overloaded I could use the existing keyword. I don't currently see it capable of doing that.